File: fastapi_app/examples/Python/api-assistant-chatbot-post.py
import requests

user_id = '123'
your_company = 'your_company'
endpoint_url = f'https://{your_company}.com/api/assistant/chatbot/{user_id}'

payload = {
  "user_input": "string",
}

response = requests.post(endpoint_url, json=payload)
if response.status_code == 200:
    print('Response:', response.json())
else:
    print('Error:', response.text)

File: fastapi_app/examples/NodeJS/api-assistant-chatbot-post.js
const axios = require('axios');

const userId = '123'; // Replace with actual user ID
const endpointUrl = 'https://{your_company}.com/api/assistant/chatbot/${userId}';

const payload = {
  user_input: 'string',
};

axios.post(endpointUrl, payload)
  .then((response) => {
    console.log('Response:', response.data);
  })
  .catch((error) => {
    console.error('Error:', error);
  });